### PR TITLE
Adds 1911 ammo & gun crates. Makes the autolathe and said guns use non-stamina damage bullets.

### PR DESCRIPTION
**Pull request by PopeDaveThe3th**
## What Does This PR Do
Adds 1911 gun and ammo crates to cargo, as contraband under the Security tab.

Swaps the .45 ammo in the autolathe and said crates to the non-stamina damage version.

## Why It's Good For The Game
M1911s are really damn cool. 

For a mildly better reason, this gives a neat sidegrade to the WT for officers who'd rather be RP'ing 1980s Miami Vice. Also useful for traitors or QMs who want to arm their goons with the finest pistol America has to offer.

## Testing
- Loaded in
- Multitooled the cargo console board
- Ordered crates 
- Opened crates
- Acquired firepower
- Shot tajaran (stamina damage is gone)
- Printed .45 ammo
- Shot tajaran again (still no stamina damage)

## Changelog
:cl:
add: Added 1911 ammunition and gun crates to cargo
tweak: Swapped the .45 ammo in the autolathe and the 1911 to non-stamina damage variant 
/:cl:

### CODE REVIEW

File: code/modules/supply/supply_packs/pack_security.dm [hunk 1]
```diff
@@ -256,6 +256,27 @@
 	cost = 500
 	containername = "auto rifle ammo crate"
 
+/datum/supply_packs/security/armory/m1911
+	name = "M1911 Pistol Crate"
+	contains = list(/obj/item/gun/projectile/automatic/pistol/m1911,
+					/obj/item/gun/projectile/automatic/pistol/m1911,
+					/obj/item/gun/projectile/automatic/pistol/m1911,)
```
Comment (Henri215) [inline code/modules/supply/supply_packs/pack_security.dm:263]:
```suggestion
					/obj/item/gun/projectile/automatic/pistol/m1911,
					/obj/item/gun/projectile/automatic/pistol/m1911)
```

File: code/modules/supply/supply_packs/pack_security.dm [hunk 2]
```diff
@@ -256,6 +256,27 @@
...
+	contraband = TRUE
+
+/datum/supply_packs/security/armory/m1911ammo
+	name = "M1911 Ammo Crate"
+	contains = list(/obj/item/ammo_box/magazine/m45,
+					/obj/item/ammo_box/magazine/m45,
+					/obj/item/ammo_box/magazine/m45,
+					/obj/item/ammo_box/magazine/m45,
+					/obj/item/ammo_box/magazine/m45,
+					/obj/item/ammo_box/magazine/m45,)
```
Comment (Henri215) [inline code/modules/supply/supply_packs/pack_security.dm:275]:
```suggestion
					/obj/item/ammo_box/magazine/m45,
					/obj/item/ammo_box/magazine/m45)
```

**Comment by Probablyjay19** [comment]
Also cool because it gives .45 ACP boxes, something that's been in the autolathe as long as I can remember, a purpose.

**Comment by Gatchapod** [comment]
I highly question balance here.
500 credits for 3 non-suppressable stechkins is extremely cheap. One player can afford that crate 30 minutes into the round without any work involved.
To get inside comfortably, you will likely want an emag for 30 TC. By spending 500 credits and 30 TC, you get access to 3 concealable pistols and the most versatile tool in the uplink.
Let us hypothetically say you're a human and you can get a holo-cigar for 10 TC. This means that for 40 TC, you get an emag and firepower of a .357 revolver (with a minor drawback of more complex reloading).
This leaves you with 60 more TC, which can afford you some extremely powerful gear like stealth chip, adrenals chip, martial art, MODsuit with thermals or holoparasite.

Overall, I feel like this gives cargo traitors way too much firepower.

**Comment by Gatchapod** [comment]
> 
> Thoughts?

This slows down the process but doesn't exactly prevent reaching some lucky surplus levels of firepower, without RNG involved.
Overall, I did that math largely to point out that relatively cheap concealable ballistics in cargo can be problematic if min-maxing is applied and this PR might be upsetting the balance too much to make the cool factor make up for it.

While I personally still worry it might be a bad idea altogether, regardless of numbers, balance team can judge it better. I am probably a bit out of touch in terms of combat, I feel.

**Comment by lewcc** [comment]
Feels to me like this gun crate might be better under an emag vs the easily accessible contraband section? 

That would keep it in traitor-only hands (or security's hands, if traitors emag it), and may justify the ease of access for these guns. Also, since it's relatively loud and traceable, it might be balanced out there?

**Comment by PopeDaveThe3th** [comment]
> Feels to me like this gun crate might be better under an emag vs the easily accessible contraband section? 
> 
> 
> 
> That would keep it in traitor-only hands (or security's hands, if traitors emag it), and may justify the ease of access for these guns. Also, since it's relatively loud and traceable, it might be balanced out there?

My intention was for this to be an expensive sidegrade to the WT for security, but if design team wants it to be emag-only that's fine too.

**Comment by farie82** [comment]
Sorry for responding late to this pull request.
The teams have voted against this PR being merged for the reason that NTs weaponry should mainly be lasers/energy based weapons instead of balistics. This PR would go against that vision.

**Comment by PopeDaveThe3th** [comment]
> Sorry for responding late to this pull request. The teams have voted against this PR being merged for the reason that NTs weaponry should mainly be lasers/energy based weapons instead of balistics. This PR would go against that vision.

Damn.

**Pull request closed without merging**